cli/src/commands/fix.rs

```diff
@@ -271,8 +271,19 @@ fn run_tool(
         s.spawn(move || {
             let mut buf: Vec<u8> = Vec::new();
             stderr.read_to_end(&mut buf).ok();
+            let prefix = {
+                let path_converter = RepoPathUiConverter::Fs {
+                    cwd: "".into(),
+                    base: "".into(),
+                };
```

Comment:
For UI message, I think we should use `workspace_command.path_converter()` or `.format_file_path()`.